lib/teleterm/daemon/daemon.go

```diff
@@ -175,6 +184,16 @@ func (s *Service) createGateway(ctx context.Context, params CreateGatewayParams)
 		LocalPort:             params.LocalPort,
 		CLICommandProvider:    cliCommandProvider,
 		TCPPortAllocator:      s.cfg.TCPPortAllocator,
+		OnExpiredCert: func(ctx context.Context, gateway *gateway.Gateway) error {
```

Comment:
Maybe make it into a method on `*Service`? And then `OnExpiredCert: s.onExpiredCert`